### ISSUE TITLE
Show Bricklink Catalog Info can be set to a default country  ( Ctrl B C )

### ISSUE THREAD

**Issue by Brick546** [rgriebl/brickstore#489]
Hi Guys, 

When we select an item part and click on " Show Bricklink Catalog Info " Bricklink defaults to all countries. I am located in Australia which has its own unique pricing structure that is very different to the rest of the world.  This means that every time "Ctrl B C" is selected I then have to scroll down, click country and click search. Might not sound like much work but doing this 100's of times per set is a killer. Bricklink already has my country has the default , when I select seller location and click search it then adds this syntax to the URL        %22loc%22:%22AU%22   .  

My question is if there is any current way ( including user script area ) that I can add this as a default "Ctrl B C"  search criteria. 

If not I can still see this being relevant to a lot of users where they just want to look at local prices especially with the way freight prices are heading,

Cheers,
Adam

**Comment by BrickSouth** [comment]
I agree especially with your last comment. I think there might be similar issues already open, but something I have been wondering about is if it's possible to set prices based on country. It looks like currently the price guide in Brickstore is based on the "Anywhere" option in the Bricklink price guide settings. Would be great if we could change it to our specific country.

**Comment by Brick546** [comment]
Thanks for your feedback. Similar topic but ever so slightly different. I'm not chasing the average cost for my Country ( would be nice but not essential for me ) I'm more about seeing the first page ( cheapest entries ) for my country by default. Average is a good guide but quite often very misleading if you want to be competitively priced without being to cheap.

**Comment by jjcchh754** [comment]
Hello. I would def love have a feature in brickstore where I can receive local prices for my country sweden. Thanks for a super nice product.
